src/tests/Interop/NativeLibrary/API/NativeLibraryTests.cs

```diff
@@ -46,6 +46,23 @@ public void LoadLibraryOnInvalidFile_NameOnly()
         EXPECT(TryLoadLibrary_NameOnly(libName), TestResult.ReturnFailure);
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Why Windows-only? Shouldn't non-Windows also throw on an invalid relative path and work with a valid one?